src/data/services/EnterpriseDataApiService.js

```diff
@@ -204,6 +204,14 @@ class EnterpriseDataApiService {
     const url = `${EnterpriseDataApiService.enterpriseAdminAnalyticsV2BaseUrl}${enterpriseId}/${chartUrl}?${new URLSearchParams(options).toString()}`;
     return EnterpriseDataApiService.apiClient().get(url);
   }
+
+  static getEnrollmentsCSVDownloadURL(enterpriseId, options) {
```

Comment:
Would you please explain the rational behind this service function?

Reply:
Getting the download URL is tied to backend service as it depends on the deployment URL of the service. So, I placed the utility function in here.